src/layers/domain/core/cpm_system_id/tests/test_cpm_system_id_v1.py

```diff
@@ -1,5 +1,21 @@
+import os
+from pathlib import Path
+
 import pytest
 from domain.core.cpm_system_id import AsidId, PartyKeyId, ProductId
+from event.json import json_load
+
+PATH_TO_CPM_SYSTEM_IDS = Path(__file__).parent.parent
+PRODUCT_IDS_GENERATED_FILE = f"{PATH_TO_CPM_SYSTEM_IDS}/generated_ids/product_ids.json"
+generated_product_ids = set()
+
+
+@pytest.fixture(scope="module", autouse=True)
+def _get_generated_ids():
+    global generated_product_ids
```

Comment:
```suggestion
generated_product_ids = set()


@pytest.fixture(scope="module")
def generated_product_ids():
```

and then use the `generated_product_ids` fixture in the arg of the required test?

Reply:
Either way the fixture is called once, but ok.